src/lib/flex-attributes.scss

```diff
@@ -20,12 +44,36 @@
 	@each $name, $_ in $flex-layout-media-queries {
 		@include flex-layout-media($name) {
 			*[data-fx-flex-#{$name}] {
+				@include flex-auto-properties;
+			}
+
+			*[data-fx-flex-#{$name}="auto"] {
```

Comment:
Is this correct? `data-fx-flex-XX` will add `flex-auto-properties` now and `data-fx-flex-XX="auto"` will add `flex-properties`.

Reply:
Ow probably a copy paste error. I'll fix it.